orders/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+class Bidding(models.Model):
+    created_at     = models.DateTimeField(auto_now_add=True)
+    updated_at     = models.DateTimeField(auto_now_add=True)
+    expired_within = models.ForeignKey('ExpiredWithin', on_delete=models.SET_NULL, null=True)
+    is_seller      = models.BooleanField()
+    user           = models.ForeignKey('users.User', on_delete=models.SET_NULL, null=True)
+    product        = models.ForeignKey('products.Product', on_delete=models.SET_NULL, null=True)
+    bid_price      = models.DecimalField(max_digits=18, decimal_places=2)
```

Comment:
이미 bidding table에 있는 속성이므로 price만으로 의미 전달이 충분한 것 같습니다!

```suggestion
    price      = models.DecimalField(max_digits=18, decimal_places=2)
```

Reply:
넵! 수정했습니다